src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1551,7 +1554,8 @@ private void deleteViewAcls(
     viewsToDelete.forEach(
         tableName -> logger.info("Deleting ACLs for view " + snapshotName + "." + tableName));
     List<Acl> acls = convertToViewAcls(snapshotProjectId, snapshotName, viewsToDelete);
-    bigQueryProject.removeDatasetAcls(datasetBqDatasetName, acls);
+    bigQuerySnapshotProject.removeDatasetAcls(datasetBqDatasetName, acls);
```

Comment:
What acls does this remove? When we create a snapshot view, I see we add the view acls to the dataset but I don't see where we do this for the snapshot: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java#L396

Reply:
Hmm. Maybe we don't need to delete from the snapshot, only the dataset.

Reply:
Yup. Confirmed tests pass with just the dataset acls being removed. Thanks!

Reply:
Awesome! Thanks for confirming that!